include/gatekeeper_fib.h

```diff
@@ -19,6 +19,9 @@
 #ifndef _GATEKEEPER_GK_FIB_H_
 #define _GATEKEEPER_GK_FIB_H_
 
+#include <lualib.h>
+#include <lauxlib.h>
```

Comment:
Once `get_fib_dump_reg()` goes away, we should likely move these includes to `gk/fib.c`.